third_party/mkl/BUILD

```
@@ -47,13 +47,20 @@ filegroup(
     visibility = ["//visibility:public"],
 )
 
+cc_library(
+    name = "mkl_libs_linux",
+    srcs = [
+        "@llvm-openmp//:libiomp5.so",
+    ],
+    visibility = ["//visibility:public"],
+)
+
 cc_library(
     name = "intel_binary_blob",
     visibility = ["//visibility:public"],
     deps = select({
         "@org_tensorflow//tensorflow:linux_x86_64": [
-            "@mkl_linux//:mkl_headers",
-            "@mkl_linux//:mkl_libs_linux",
+            ":mkl_libs_linux",
```

Or we can just link to the llvm-openmp target directly.
```suggestion
            "@llvm-openmp//:libiomp5",
```

---

I tried it and got errors I could not resolve. Once the upcoming Wnidows PR is also merged, I will go back and clean up the old mkl builds and try to fix it then.

---

I think the error was because your `libiomp5` target was a `cc_binary` and not `cc_library`. Please try again with `cc_library`.

---

Let's keep using `mkl_libs_linux` for now. Could you please add a TODO to change to link directly to `@llvm-openmp//:libiomp5` later? Thank you very much!